pages/docs/concepts/asyncapi-document/adding_messages.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Adding Messages
+weight: 140
+---
+
+[Messages](../message.md) in AsyncAPI represent the data being transmitted between your systems. They always include a payload, which is the data the application is sending or receiving, and they may also include headers, which are used to provide additional context or information about the message exchange.
+
+```mermaid
+graph TD;
+    A[AsyncAPI Message] --> B[Headers]
+    A --> C[Payload]
+    B --> D[Header Content]
+    C --> E[Payload Content]
+
+    style A fill:#47BCEE,stroke:#47BCEE;
+```
+
+Here is an example of a simple message:
+
+```yml
+channels:
```

Comment:
Should this doc target version 3 of the spec instead? @alequetzalli

Reply:
So all out GSoD work is for version 3. 
It seems my code snippet has some error. Can you please give me some instinct on it?

Reply:
@jonaslagoni wrote a few examples in this article: https://github.com/asyncapi/website/pull/2071. Does that help? If not, I'm happy to jump on a call and explain the new syntax if needed.